### ISSUE TITLE
Link text has insufficient color contrast

### ISSUE THREAD

**Issue by terrill** [uweb/uw-2014#33]
Link text (#08d on white) fails to meet WCAG 2.0 contrast requirements for compliance at Level AA. The current combination has a contrast ratio of 3.7731, whereas Level AA compliance requires at least 4.5. To meet the requirement with minimal impact, change link color to **#0074bb**.

**Comment by nwinkelbauer** [comment]
addressed by PR https://github.com/uweb/uw-2014/pull/47
